Bucketeer/Sources/Internal/Evaluation/EvaluationSQLDao.swift

```diff
@@ -16,11 +18,13 @@ final class EvaluationDaoImpl: EvaluationDao {
         try db.select(EvaluationEntity(), conditions: [.equal(column: "userId", value: userId)])
     }
 
-    func deleteAllAndInsert(userId: String, evaluations: [Evaluation]) throws {
-        let entities = try evaluations.map {
-            try EvaluationEntity(model: $0)
-        }
+    func deleteAll(userId: String) throws {
         try db.delete(EvaluationEntity(), condition: .equal(column: "userId", value: userId))
-        try db.insert(entities)
+    }
+
+    func deleteByIds(_ ids: [String]) throws {
+        for id in ids {
+            try db.delete(EvaluationEntity(), condition: .equal(column: "id", value: id))
```

Comment:
This should also use transaction

Reply:
fixed

Reply:
But I am using transaction from higher level (EvaluationStorage)